### ISSUE TITLE
Fix FPKI Alt Text

### ISSUE THREAD

**Issue by JillTunick** [GSA/playbooks-wdsv2#449]
Fix the alt text for the gifs and images on these pages:
https://playbooks.idmanagement.gov/fpki/common/obtain-and-verify/
https://playbooks.idmanagement.gov/fpki/common/distribute-os/

The AMP tool has flagged alt text violations for the gifs and images on these pages. Sample violation note: 

This IMG has a suspicious calculated accessible name value of: a .gif that shows the distribution and verification steps performed using microsoft certutil
